### PR TITLE
OCPBUGS#3646: service account is used to admit the pods

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.9+

Issue: https://issues.redhat.com/browse/OCPBUGS-3646

Link to docs preview: https://56483--docspreview.netlify.app/openshift-enterprise/latest/authentication/managing-security-context-constraints.html#admission_configuring-internal-oauth

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by kcarmichael08 — COMMENTED** [review]
Hi Servesha, can you please put the HTML preview link in the body of the PR? 

I had a small suggestion, based on the ticket and the slides referenced in the ticket, but since it has been reviewed by QE, I don't think it's a blocker to leave it as is. (Also I guess the QE box needs to be checked in the body of the PR.)

File: modules/security-context-constraints-about.adoc [hunk 1]
```diff
@@ -380,6 +380,11 @@ user identity and groups that the user belongs to. Additionally, if the pod
 specifies a service account, the set of allowable SCCs includes any constraints
 accessible to the service account.
 
+[NOTE]
+====
+When you create a deployment, only the service account is used to find the SCCs and is used to admit the pods when they get created.
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/security-context-constraints-about.adoc]: Addressed

**Comment by xenolinux** [comment]
> One other thing - just verifying that this applies to 4.9 and 4.10 since the ticket says 4.11.

QE confirmed that the change applies to 4.9 and 4.10 as well so we can backport it to 4.9 and 4.10.

**Comment by mburke5678** [comment]
@xenolinux  ^^

**Pull request merged**